md-rest-api/src/main/java/com/wipro/ats/bdre/md/rest/RestWrapper.java

```diff
@@ -89,13 +87,15 @@ public int getTotalRecordCount() {
                 for (Object obj : c) {
                     try {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Refactor this code to not nest more than 3 if/for/while/switch/try statements. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS134)